M17_spec.tex

```diff
@@ -1160,162 +1158,224 @@ \subsection{TYPE Field}
 		\diagbox{Byte}{Bit} & 7 & 6 & 5 & 4 & 3 & 2 & 1 & 0 \\
 		\Hline[tikz=very thick]
 		0 &
-			\multicolumn{4}{c}{\textit{Reserved}} &
-			\parbox{3em}{\centering Signed Stream} &
-			\multicolumn{3}{c}{Channel Access Number\ldots} \\
+			\multicolumn{4}{c}{Payload Type} &
+			\multicolumn{3}{c}{Encryption} &
+			\parbox{3em}{Signed} \\
 		\hline
 		1 &
-			\ldots &
-			\multicolumn{2}{c}{\parbox{6em}{\centering Encryption Subtype}} &
-			\multicolumn{2}{c}{\parbox{6em}{\centering Encryption Type}} &
-			\multicolumn{2}{c}{\parbox{6em}{\centering Data Type}} &
-			\parbox{3em}{\centering Packet/ Stream} \\
+			\multicolumn{4}{c}{Meta Data} &
+			\multicolumn{4}{c}{Channel Access Number} \\
 		\hline
 	\end{NiceTabular}
 	\normalsize
 	\caption{Stream LSF TYPE Layout}
 \end{table}
 
+\subsection{Payload}
```

Comment:
It would be nice if the heading titles matched the text in the table above. So if we make the change I suggested above, this would become 'Payload Contents'.